wasm/store_test.go

```diff
@@ -0,0 +1,22 @@
+package wasm
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetModuleInstance(t *testing.T) {
+	name := "test"
+
+	// 'jit' and 'wazeroir' package cannot be used because of circular import.
+	// Here we will use 'nil' instead. Should we have an Engine for testing?
+	s := NewStore(nil)
+
+	m1 := s.getModuleInstance(name)
+	assert.Equal(t, m1, s.ModuleInstances[name])
```

Comment:
nit is we prefer require vs assert as getting errors early helps avoid test panics

Reply:
Thanks for the feedback!
Fixed in a66e636.